projects/wave-core-new/src/lib/project/project.service.ts

```diff
@@ -598,6 +589,28 @@ export class ProjectService {
         return subject.asObservable();
     }
 
+    /**
+     * Remove all plots from the current project.
+     */
+    clearPlots(): Observable<void> {
+        const subject = new Subject<void>();
```

Comment:
what is this needed for?

Reply:
What is it good for? Absolutely nothing.

17498ce